core/audits/uses-rel-preconnect.js

```diff
@@ -235,7 +239,7 @@ class UsesRelPreconnectAudit extends Audit {
       {key: 'wastedMs', valueType: 'timespanMs', label: str_(i18n.UIStrings.columnWastedMs)},
     ];
 
-    const details = Audit.makeOpportunityDetails(headings, results, maxWasted);
+    const details = Audit.makeOpportunityDetails(headings, results, null, maxWasted);
```

Comment:
Can you comment why we pass `null` here

Reply:
`uses-rel-preconnect` audit has access to complete URLs from network records a bit earlier in the code (see L212). Entity lookup is done at that time. The URLs that we have at this point (and passed to opportunity generation) have been trimmed down to just their security origins for preconnect recommendation. So we avoid a second entity lookup for two reasons, (1) it's redundant, (2) since security origins might not be a part network record URLs, lookups are not guaranteed to succeed. 

That said, we could pass `classifiedEntities` here and it would not have any effect. `makeOpportunityDetails` gives preference to `item.entity` that already exists on the object.